cmd/manager/main.go

```diff
@@ -138,7 +174,8 @@ func main() {
 		RenewDeadline:          &leaderElectionRenewDeadline,
 		RetryPeriod:            &leaderElectionRetryPeriod,
 		WebhookServer: webhook.NewServer(webhook.Options{
-			Port: 9443,
+			Port:    9443,
```

Comment:
we might not need webhook server as well, lets check and address this in followup as well.